server/service/software_installers.go

```diff
@@ -79,6 +80,11 @@ func (uploadSoftwareInstallerRequest) DecodeRequest(ctx context.Context, r *http
 		decoded.PostInstallScript = val[0]
 	}
 
+	val, ok = r.MultipartForm.Value["self_service"]
+	if ok && len(val) > 0 && val[0] != "" {
```

Comment:
That would make it `true` even if explicitly set to `false`, I think once the check for presence of a value is done, it should still be parsed for truthy/falsy? (maybe using `strconv.ParseBool`)

Reply:
That is true, I can use that 👍